### DISCUSSION TITLE
Essentials - 4 items on full card

### DISCUSSION THREAD

**Discussion by EddieSpaghettie** [slipx06/sunsynk-power-flow-card#415] [Lux]
![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/58969883/ee1a2eff-3e61-47fb-bdda-bb794d913ce2)

If one doesnt have AUX items then id like 4 essentials to be available on the full card :

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/58969883/eb2a262e-332b-4fe9-98f6-87805a00184f)

there is space cause in the light view it can show

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/58969883/e231741a-563e-41d4-9597-3d6e329d7ea8)

**Comment by slipx06** [comment]
Working on it

**Comment by slipx06** [comment]
Added to https://github.com/slipx06/sunsynk-power-flow-card/releases/tag/v4.36.0

**Reply by EddieSpaghettie** [reply]
Thank you

**Comment by EddieSpaghettie** [comment]
I see its working. Awesomeness!

 



 

 


Edwin Smith

	

  +27837844112
  ***@***.*** ***@***.***> 


 <https://www.facebook.com/edseds>   <https://www.youtube.com/user/CodeTwoChannel>   <https://www.linkedin.com/in/edwin-smith-42b91831/>  

​Im not just a biker but a Father and a Husband too!

**Reply by Roving-Ronin** [reply]
@EddieSpaghettie any chance you could post up a picture of what this looks like ? Wondering is this 4 addition (i.e 8 total) of what essential load and layout? Thanks.

**Reply by EddieSpaghettie** [reply]
Im using 4 essentials now, could only add 2 before and 2 aux. so disabled aux and put 4 essentials : 

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/58969883/5006623f-3f5d-4656-a090-1c499ffe3d4f)

**Reply by Roving-Ronin** [reply]
OK so it provides an 'official' place to show up to 4 items, but lose the ability to get a total energy (kWh) total for the items, when repurposing the AUX to display them (instead of a generator etc). e.g.

![image](https://github.com/slipx06/sunsynk-power-flow-card/assets/108674933/e776162a-9016-4611-83ff-9fb94d031981)

Trying to remember, but I think there was a change last month that does allow somehow fitting a secondary reading next to each load (assuming it could be used for kWh for each individual item, but no sum total for them, only the essential / whole house).
